src/assets/styles/theme.scss

```diff
@@ -81,4 +81,5 @@ $violet_80: #140D19;
 
 $light-blue-80: #7CADFF;
 
+$violet_50: #D7CFFC;
```

Comment:
jeżeli to juz jest violet inny wyżej to ten z mniejszyn numerem powinien być nad nim